migrations/20170622165827-ports-migration.js

```diff
@@ -0,0 +1,130 @@
+module.exports = {
```

Comment:
so you only want this logic if we are running the migration? Can you elaborate on that a little for me?